website/blog/2023-01-27-Introducing-native-support-for-Apache-Hudi-Delta-Lake-Apache-Iceberg-on-AWS-Glue-for-Apache-Spark.mdx

```diff
@@ -5,8 +5,8 @@ authors:
 category: blog
 image: /assets/images/blog/0127-introducing-native-support-hudi-aws-glue.png
 tags:
-- apache hudi
```

Comment:
What is the criteria of adding the "apache hudi" tag? I assume all blogs are on Apache Hudi?

Reply:
Couple reasons. 
1. We use one tag for marking the source of the article. For example if it came from an individual account in medium we use medium. Similarly If a blog from amazon or uber we tag it as amazon or uber. For blogs coming natively from apache hudi website we tag that way. These are documented in Readme file.
2. Whenever a reference to similar oss projects come into picture we also tag apache hudi for parity. 
Other than that it is hard to keep it consistent across different people tagging the blogs. In this Pr I changed the tags from hudi -> apache hudi for keeping it consistent.

Reply:
Sounds good.  Thanks for the clarification!